filament/include/filament/View.h

```diff
@@ -571,6 +572,115 @@ class UTILS_PUBLIC View : public FilamentAPI {
     //! debugging: returns a Camera from the point of view of *the* dominant directional light used for shadowing.
     Camera const* getDirectionalLightCamera() const noexcept;
 
+
+    /** Result of a picking query */
+    struct PickingQueryResult {
+        utils::Entity renderable{};     //! RenderableManager Entity at the queried coordinates
+        float depth{};                  //! Depth buffer value (1 (near plane) to 0 (infinity))
+        uint32_t reserved1{};
+        uint32_t reserved2{};
+        /**
+         * screen space coordinates in GL convention, this can be used to compute the view or
+         * world space position of the picking hit. For e.g.:
+         *   clip_space_position  = (fragCoords.xy / viewport.wh, fragCoords.z) * 2.0 - 1.0
+         *   view_space_position  = inverse(projection) * clip_space_position
+         *   world_space_position = model * view_space_position
```

Comment:
Don't you mean `inverse(view_space_position)` here? And you don't need the model matrix?

Reply:
`view_space_position` is a `vec3` so `inverse` of that is not possible.
The world space position is the `modelMatrix * viewSpacePosition`. Of course, `the modelMatrix` is the `inverse(viewMatrix)`. But `Camera` has a `getModelMatrix()`, so that's why I used this convention.

Reply:
Ah here you mean model matrix of the camera, I thought you meant of a renderable. It makes sense in hindsight but maybe clarify. In this case, yes, that's the correct formula (and I mean inverse(viewMatrix) not of the view position, sorry).